src/components/apphost.js

```diff
@@ -220,7 +220,7 @@ define(["appSettings", "browser", "events", "htmlMediaHelper"], function(appSett
             }
         },
         deviceIconUrl: function() {
-            return browser.edgeUwp, browser.edgeUwp ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/windowsrt.png" : browser.opera || browser.operaTv ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/opera.png" : browser.orsay || browser.tizen ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/samsungtv.png" : browser.web0s ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/lgtv.png" : browser.ps4 ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/ps4.png" : browser.chromecast ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/chromecast.png" : browser.chrome ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/chrome.png" : browser.edge ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/edge.png" : browser.firefox ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/firefox.png" : browser.msie ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/internetexplorer.png" : browser.safari ? "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/safari.png" : "https://github.com/MediaBrowser/Emby.Resources/raw/master/images/devices/html5.png"
+            return browser.edgeUwp, browser.edgeUwp ? "/web/css/images/devices/windowsrt.png" : browser.opera || browser.operaTv ? "/web/css/images/devices/opera.png" : browser.orsay || browser.tizen ? "/web/css/images/devices/samsungtv.png" : browser.web0s ? "/web/css/images/devices/lgtv.png" : browser.ps4 ? "/web/css/images/devices/ps4.png" : browser.chromecast ? "/web/css/images/devices/chromecast.png" : browser.chrome ? "/web/css/images/devices/chrome.png" : browser.edge ? "/web/css/images/devices/edge.png" : browser.firefox ? "/web/css/images/devices/firefox.png" : browser.msie ? "/web/css/images/devices/internetexplorer.png" : browser.safari ? "/web/css/images/devices/safari.png" : "/web/css/images/devices/html5.png"
```

Comment:
Why `css/images`?

Reply:
These images are only attached to the CSS element "cardBuilder".

I have no qualms with a better position for the images, just put them somewhere that kinda sorta made sense.

If you got one, let me know!

Reply:
Would this be more readable as a switch/case construct? More than one ternary operator in a return just is too much IMHO.

Reply:
This is still uglified code. This entire file has not been deuglified and I didn't do it as part of this PR as its such a simple change compared to actually changing logic.

Reply:
css seems like the wrong place tbh

Reply:
Well, I could move it up a folder level and have it in a folder named `images` but then, instead of organizing, we have just added yet another folder with images in it.

Reply:
I would say that putting them under `images` next to `css` suits me more than shoving them into `css`. Or maybe rename `css` to `assets` (but that's a less-preferable way for me).

Reply:
Also certainly, even if all other file is uglified, make this a switch.

Plus with those absolute URLs you're effectively killing the possibility to host this under URL prefix :( There's a `baseUrl()` function that _might_ serve as a starter for URLs here.